src/app/components/server/BreachLogo.tsx

```diff
@@ -14,12 +14,20 @@ import { Breach } from "../../(nextjs_migration)/(authenticated)/user/breaches/b
  * @param {Map<string, string>} props.logos Map of URLs to logos indexed by the domain name of the respective company
  * @returns {string} HTML for a breach logo (either an `img`, or a `span.breach-logo` containing the breached company's first letter)
  */
-export function BreachLogo(props: { breach: Breach; logos: LogoMap }) {
+
+export type Props = {
+  breach: Breach;
+  logos: LogoMap;
+  htmlTags?: boolean;
```

Comment:
Could you add a comment here describing why this is needed? (And that it can be removed once everything is proper React - assuming that that's the case?)

Reply:
Added the following comment in [e3f0628](https://github.com/mozilla/blurts-server/pull/3137/commits/e3f0628d0cbc5b90255404d9fa5709bb0b694cfd):

> If a component imports `react-dom/server` directly Next.js throws a compilation error `ReactServerComponentsError` for perf and security reasons.

Reply:
Done in https://github.com/mozilla/blurts-server/pull/3137/commits/e3f0628d0cbc5b90255404d9fa5709bb0b694cfd.